### PR TITLE
[zh-cn]Sync garbage-collection.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Synchronized some upstream changes and optimized some Chinese displays
```yaml
content/zh-cn/docs/concepts/architecture/garbage-collection.md
```

### CODE REVIEW

**Review by Sea-n — COMMENTED** [review]
Thanks for purposing new wording.
But I'm not sure if it help the document better or not.

In the other word, what value did this PR bring to k/website repo?

**Comment by yanrongshi** [comment]
> Thanks for purposing new wording. But I'm not sure if it help the document better or not.
> 
> In the other word, what value did this PR bring to k/website repo?

我认为老这个字过于哩语，所以依据上下文进行了新的说明，此外 `垃圾搜集` 和 `垃圾收集` 应当是两个概念，当然也有可能是我的理解错误

**Comment by windsonsea** [comment]
> > Thanks for purposing new wording. But I'm not sure if it help the document better or not.
> > In the other word, what value did this PR bring to k/website repo?
> 
> 我认为老这个字过于哩语，所以依据上下文进行了新的说明，此外 `垃圾搜集` 和 `垃圾收集` 应当是两个概念，当然也有可能是我的理解错误

You can give some desc text, such as:

- fixed typos
- replaced italic with bold in the translated text
- 
So everyone can easily know what you have provided for the value.
image

**Comment by windsonsea** [comment]
Please also change the content mentioned in #35742 for this file.

**Comment by windsonsea** [comment]
You can try to run the following commands to rebase:
```
git checkout main
git pull -r
git checkout zh-cn]Sync-garbage-collection.md
git rebase main
```
Manually resolve the conflicts one by one by following the on-screen instructions, and then run the following commands:
```
git add .
git rebase --continue
git push origin zh-cn]Sync-garbage-collection.md
```

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ allows the clean up of resources like the following:
 * [Node Lease objects](/docs/concepts/architecture/nodes/#heartbeats)
 -->
 * [终止的 Pod](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+=======
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:36]: rebase 出来的多余行

**Comment by windsonsea** [comment]
Please push again to refresh the pipeline while `/retest` doesn't work.

**Pull request merged**